examples/trials/cifar-adaptdl/main.py

```diff
@@ -155,22 +144,28 @@ def valid(epoch):
         stats["accuracy"] = stats["correct"] / stats["total"]
         writer.add_scalar("Loss/Valid", stats["loss_avg"], epoch)
         writer.add_scalar("Accuracy/Valid", stats["accuracy"], epoch)
-        if IS_CHIEF:
-            nni.report_intermediate_result(stats["loss_avg"], accum=stats)
+
+        if adaptdl.env.replica_rank() == 0:
+            nni.report_intermediate_result(stats["accuracy"], accum=stats)
+
         print("Valid:", stats)
-        return stats["loss_avg"]
+        return stats["accuracy"]
 
 
+# TODO BE-12547: Rename ADAPTDLCTL_TENSORBOARD_LOGDIR to ADAPTDL_TENSORBOARD_LOGDIR
 tensorboard_dir = os.path.join(
     os.getenv("ADAPTDLCTL_TENSORBOARD_LOGDIR", "/adaptdl/tensorboard"),
     os.getenv("NNI_TRIAL_JOB_ID", "cifar-adaptdl")
 )
 if not os.path.exists(tensorboard_dir):
     os.makedirs(tensorboard_dir)
+
 with SummaryWriter(tensorboard_dir) as writer:
-    for epoch in et.remaining_epochs_until(args.epochs):
+    acc = 0
```

Comment:
"acc=0" can be removed.

Reply:
This is more of the code style preference. In most other languages, we must declare a variable first. So this line is to conform with this tradition. Besides, if we don't declare `acc=0`, a lint error will be raised in my IDE ("possible unbound variables").